docs/resources/task.md

```diff
@@ -122,9 +122,11 @@ The Iterative Provider offers some common machine types (medium, large, and extr
 
 | Type      | Minimum CPU cores | Minimum RAM | GPU                 |
 | :-------- | ----------------: | ----------: | :------------------ |
+| `s`       |                 1 |        1 GB | -                   |
 | `m`       |                 8 |       16 GB | -                   |
 | `l`       |                32 |       64 GB | -                   |
 | `xl`      |                64 |      128 GB | -                   |
+| `s+t4`    |                 1 |        1 GB | 1 NVIDIA Tesla T4   |
```

Comment:
```suggestion
| `m+t4`    |                 4 |       16 GB | 1 NVIDIA Tesla T4   |
```

or even

```suggestion
| `s+k80`   |                 2 |       12 GB | 1 NVIDIA Tesla K80  |
| `m+t4`    |                 4 |       16 GB | 1 NVIDIA Tesla T4   |
```

Reply:
> _On some providers_

Here is where I stop reading. 🐉

Users that are dumb enough not to check their provider's documentation should be able to use our small, curated set of defaults on every provider.

If they have to figure out why some allegedly generic machine type doesn't work in their cloud provider, something is wrong with us.

Reply:
I would provide only a small set of “common” machine types for demonstration purposes; it's “fine” 🔥🐶 to add `m+t4` for demonstration & testing purposes. 

For now at least, users should be responsible for navigating their cloud provider's documentation if they want other instance types.